test/vcr/vcr_cassettes/service_bus/test_0001_should_be_able_to_send_a_message_with_SAS_to_a_queue.yml

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
I was expecting key 'SB_SAS_KEY' in this file but i don't see it or i missed it somewhere. could you please double check?